src/pd/retry.rs

```diff
@@ -49,23 +49,25 @@ impl<Cl> RetryClient<Cl> {
             cluster: RwLock::new(cluster),
             connection,
             timeout,
+            last_connected: RwLock::new(Instant::now()),
```

Comment:
Could we put the last connected instant inside cluster to save a lock?

Reply:
> Could we put the last connected instant inside cluster to save a lock?

Updated